cPP_Lifecycle.adoc

```diff
@@ -0,0 +1,135 @@
+
+[NOTE]
+====
+While the document talks about cPP revisions, this covers the cPP, SD and any other related documents.
+====
+
+== Revision Classes
+There are three classes of revisions to a cPP that can be produced. The descriptions are not meant to be exhaustive, but to represent the level of changes in the document.
+
+.Revision Classes for a cPP
+[cols=".^1,.^1,.^4",options="header"]
+|===
+
+|Revision Class
+|Version Position
+|Description
+
+|Major
+|**X**.y.zz
+|Full update to the cPP, including requirements changes (addition or removal of requirements)
+
+|Minor
+|x.**Y**.zz
+|Update to the cPP, likely updating existing requirements, but not changing them (i.e. not adding/removing requirements)
+
+|Technical Decision or Errata
+|x.y.**ZZ**
+|Application of Technical Decisions to the cPP (no other changes)
+
+|===
+
+=== Revisions & Branches
+The {itc-shortname} maintains 3 primary branches for the creation of the cPP and all supporting documents. Of these branches, two will be used to maintain the current versions of the documents while the third is used for active work on new updates.
+
+.Branches & the Latest Documents
+[cols=".^1,.^1,.^4",options="header"]
+|===
+
+|Branch
+|Versions
+|Description
+
+|Master/Main
+|**X**.**Y**
+|This is always maintained at the last Major.Minor release and is the "official" published version.
+
+|Interpretation
+|x.y.**ZZ**
+|This branch will be updated for each TD that is published and is a rolling update for all approved TDs between Major.Minor updates.
+
+This branch can be used in place of Major.Minor + TDs if the validating scheme approves all TDs.
+
+|===
+
+The {itc-shortname} may approve errata updates on a periodic basis to officially roll TDs into a published release depending on the number of TDs and the time before the next full update.
+
+== Revision Transitions
+When a new revision will be published, it is immediately available for use. So if version 1.1 is released, it can be immediately used for an evaluation. To ensure a common direction and basis for evaluation within the community, the release of a new cPP revision means that the previous revisions must be retired from active use.
+
+The transition period is a time of overlap when multiple revisions of the cPP may be used for an active evaluation. The transition periods are determined by the type of revision being published. At the end of the transition period, the previous revision will be sunset, or retire and no longer be considered acceptable for new evaluations (evaluations already in process may still be completed).
+
+The following table specifies when the previous revision will be sunset/retired and the new revision takes precedence.
+
+.Transition Periods 
+[cols=".^1,.^1",options="header"]
+|===
+
+|Revision Class
+|Transition Period
+
+|Major
+.2+|6 months after the new revision is published
```

Comment:
Is it standard practice in other PP for major and minor updates to be on the same retiring schedule? I would have thought minor might be more rapidly adopted, presuming the changes are things that essentially could be made on the fly.

Reply:
For CC, yes, that has been the case. At one point it seemed like it would not be (a few years ago), but the reality is that the pace of work is slow enough (and the time it can take for vendors to plan new versions) that shorter periods on the minor releases is likely to cause problems. NIAP, since they produce a lot of the PPs today, has a standard 6 month sunset period after a new update is published. For simplicity I have just expected to follow this, and so far no one has thought differently (I pulled this from documents I have been working on for all iTCs to build from, and the times I have asked, there is general consensus on the 6 month timeline regardless of version).